impy/models/dpmjetIII.py

```diff
@@ -272,3 +272,25 @@ def generate_event(self):
         reject = self.lib.dt_kkinc(*self._dpmjet_tup(), kkmat=-1)
         self.lib.dtevno.nevent += 1
         return reject
+
+class DpmjetIII191(DpmjetIIIRun):
+    def __init__(self, event_kinematics, seed="random", logfname=None):
+        from impy.definitions import interaction_model_by_tag as models_dict
+        interaction_model_def = models_dict["DPMJETIII191"]       
+        super(DpmjetIIIRun, self).__init__(interaction_model_def)
```

Comment:
Since Python3, `super().__init__` is sufficient here. It makes no sense to still support Python2, which has reached end of life. Numpy dropped Python3.6 support, so the baseline to program for at this point for a scientific library is Python 3.7.

Reply:
Ok, it will be changed to `super().__init__`. Althought the intent was to call `__init__` of parent of super() (i.e. parent of `DpmjetIIIRun` in this case). However, as Anatoli noticed, it is better to call parent's `__init__` in case it will appear in the future.

Reply:
Yes. Calling `__init__` from a base class higher up the hierarchy is going against principles of OOP and likely to cause bugs in the future.